src/evaluations/evaluations.module.ts

```diff
@@ -1,7 +1,12 @@
 import {Module} from '@nestjs/common';
+import { SequelizeModule } from '@nestjs/sequelize';
```

Comment:
Idk if there is a reason for it, but this is the only import where you have spaces in the brackets.